int32 iLowH
int32 iHighH
int32 iLowS
int32 iHighS
int32 iLowV
int32 iHighV
